src/components/DiffView/index.tsx

```diff
@@ -109,6 +118,33 @@ export class DiffViewBase extends React.Component<Props> {
     }
   };
 
+  getWidgets = (hunks: Hunks) => {
+    const { linterMessages } = this.props;
+
+    return getAllHunkChanges(hunks).reduce((widgets, change) => {
+      const changeKey = getChangeKey(change);
+      const line = change.lineNumber;
+
+      let widget = null;
```

Comment:
I'd move this declaration down, closer to where it gets assigned, but that's a personal taste I guess.